src/ui/component/socialShare/view.jsx

```diff
@@ -70,7 +70,7 @@ class SocialShare extends React.PureComponent<Props> {
       <React.Fragment>
         {speechShareable && (
           <div className="card__content">
-            <label className="card__subtitle">{__('Web link')}</label>
+            <label className="help">{__('Web link')}</label>
```

Comment:
Heh, I think you changed this class because I complained, but I'd argue these aren't "help" items either.

`card__label`, `card__action-label`? or something like this? it's fine if the classes are styled exactly the same, but using `help` when it is not "help" will render future style evolution harder